scoap3/cli_harvest.py

```diff
@@ -0,0 +1,143 @@
+import json
+import logging
+from os import listdir
+from os.path import join, isdir, abspath, isfile, realpath, dirname
+
+import click
+from flask.cli import with_appcontext
+from inspire_crawler.tasks import schedule_crawl
+from invenio_oaiharvester.tasks import list_records_from_dates
+from jsonschema import validate, ValidationError, SchemaError
+
+from scoap3.modules.records.util import create_from_json
+from scoap3.modules.robotupload.util import parse_received_package
+
+logger = logging.getLogger(__name__)
+
+
+def log(msg='', level=logging.INFO, **kwargs):
+    params = ('%s=%s' % (x[0], x[1]) for x in kwargs.items())
+    logger.log(level, msg + '\t' + '\t'.join(params))
+
+
+def validate_schema(obj):
+    key = '$schema'
+
+    if key not in obj:
+        log('No schema found!', logging.WARNING, obj=obj)
+        return True
+
+    schema_path = join(dirname(realpath(__file__)), 'modules', 'records', 'jsonschemas', 'hep.json')
+    with open(schema_path, 'rt') as f:
+        schema_data = json.loads(f.read())
+
+    try:
+        validate(obj, schema_data)
+    except (ValidationError, SchemaError) as err:
+        log('validation error!', logging.ERROR, err=err)
+        return False
+
+    return True
+
+
+@click.group()
+def harvest():
+    """
+    Manual harvest commands.
+    USE AT YOUR OWN RISK. You've been warned.
+    """
+
```

Comment:
I think here we shouldn't use a `pass`, because we have an indented block with the comment. This seems to be the practice in inspire-next as well: [link](https://github.com/inspirehep/inspire-next/blob/da4ae5b28a0bee9eccea36c7426495eaa016f9b5/inspirehep/modules/fixtures/cli.py#L36)